Base/usr/share/man/man1/xargs.md

```diff
@@ -0,0 +1,33 @@
+## Name
+
+xargs - build and execute commandlines from input
+
+## Synopsis
+
+```**sh
+$ xargs [options...] command [initial-arguments...]
+```
+
+## Description
+
+`xargs` reads items from a stream, delimited by some blank character (`delimiter`), and executes the `command` as many times as there are items, with any processed `initial-arguments`, possibly followed by an item read from the input.
+Each argument in `initial-arguments` is processed by replacing any occurence of the `replacement string` with the input item, and treating the entire resulting value as _one_ argument.
+If no argument in `initial-arguments` contains the `replacement string`, an argument is added at the end of the list containing only the `replacement string`.
+`xargs` will read the items from stdin by default, and when data is read from stdin, the standard input of `command` is redirected to `/dev/null`.
+The standard input is left as-is if data is read from a file.
+
+## Options
+
+* `-I`, `--replace`: Sets the `replacement string`, this is `{}` by default
+* `-0`, `--null`: Splits the items by null characters instead of `delimiter`
+* `-d`, `--delimiter`: Sets the `delimiter`, this is a newline (`\n`) by default
+* `-v`, `--verbose`: Displays each expanded command on stderr before executing it
```

Comment:
s/stderr/standard error/

same goes for the other mentions of stdin/stdout/stderr